EmergencyRoomQueue.py

```diff
@@ -1,31 +1,111 @@
+import time
 
 
 def main():
+
+    program = True
+
+    #test
+    patient_list_queue = []
+
+    # TEMP CODE
     print("Please do some shit")
+    patient_list_queue.append(Patient(10, "Jacob"))
+    patient_list_queue[0].display_patient()
+    patient_list_queue.append(Patient(2, "Sarah"))
+    patient_list_queue[1].display_patient()
+    # TEMP CODE END
+
+    while(program != False):
+
+        userinput = str(input())
+
+
+        # WINDOW ACTIONS CONDITIONS
+        if(userinput == "Exit"):
+            program = False
+            return 0
+        # Function Call for Sort
+        if(userinput == "Sort"):
+            print("are you sure?")
+            print("yes or no")
+            queueListSort()
 
 #Function that will retrieve patient information and sort it into the linked list
-def insertPatientIntoList():
+def insertPatientIntoList(patient_list_queue, patient_name, patient_prio): #TODO

```

Comment:
Wrong use of this function. It was intended for nurses or users to input new patient data into the list and auto sort it for the doctor. 
It's current designed use is to access and update existing patient records. Which was to be done in accesspatientrecords() function.